test/Microsoft.VisualStudio.Validation.Tests/RequiresTests.cs

```diff
@@ -117,37 +117,37 @@ public void Argument_Bool_String_String_ObjectArray()
     [Fact]
     public void Argument_Bool_String_ResourceManager_String()
     {
-        Requires.Argument(true, "someParameter", TestStrings.ResourceManager, nameof(TestStrings.SomeError));
-        ArgumentException ex = Assert.Throws<ArgumentException>(() => Requires.Argument(false, "someParameter", TestStrings.ResourceManager, nameof(TestStrings.SomeError)));
+        Requires.Argument(true, "someParameter", TestStrings.ResourceManager, TestStrings.SomeError);
```

Comment:
📝 Hard for me to say whether the changes in this file are correct or not. I'm assuming they are.

Reply:
This is where I leverage the `AsConstants` metadata you were asking about.